### PR TITLE
React Without JSX

### PR DESCRIPTION
**Pull request by brilyandk**

### CODE REVIEW

File: content/docs/react-without-jsx.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ title: React Without JSX
 permalink: docs/react-without-jsx.html
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+JSX bukan persyaratan untuk menggunakan React. Menggunakan React tanpa JSX sangat cocok ketika Anda tidak ingin mengatur kompilasi di lingkungan pembangunan Anda.
```
Comment (wisn) [inline content/docs/react-without-jsx.md]:
```suggestion
JSX bukan persyaratan untuk menggunakan React. Menggunakan React tanpa JSX sangat cocok ketika Anda tidak ingin mengatur kompilasi di *build environment* Anda.
```

Kinda weird to translate `build environment`. It is more like term for me IMO.

File: content/docs/react-without-jsx.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ ReactDOM.render(
 );
 ```
 
-If you're curious to see more examples of how JSX is converted to JavaScript, you can try out [the online Babel compiler](babel://jsx-simple-example).
+Jika Anda ingin tahu lebih lanjut untuk melihat banyak contoh tentang bagaimana JSX dikonversi ke JavaScript, Anda dapat mencobanya di [the online Babel compiler](babel://jsx-simple-example).
 
-The component can either be provided as a string, or as a subclass of `React.Component`, or a plain function for stateless components.
+Komponen dapat diberikan sebagai string, atau sebagai subkelas dari `React.Component`, atau fungsi murni untuk komponen _stateless_.
```
Comment (wisn) [inline content/docs/react-without-jsx.md]:
```suggestion
Komponen dapat diberikan sebagai _string_, atau sebagai subkelas dari `React.Component`, atau fungsi murni untuk komponen _stateless_.
```

File: content/docs/react-without-jsx.md [hunk 3]
```diff
@@ -4,11 +4,11 @@ title: React Without JSX
...
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+JSX bukan persyaratan untuk menggunakan React. Menggunakan React tanpa JSX sangat cocok ketika Anda tidak ingin mengatur kompilasi di lingkungan pembangunan Anda.
 
-Each JSX element is just syntactic sugar for calling `React.createElement(component, props, ...children)`. So, anything you can do with JSX can also be done with just plain JavaScript.
+Setiap elemen hanyalah _syntactic sugar_ untuk pemanggilan  `React.createElement(component, props, ...children)`. Jadi, apa pun yang dapat Anda lakukan dengan JSX juga dapat dilakukan hanya dengan JavaScript biasa.
 
-For example, this code written with JSX:
+Misalnya, kode ini ditulis dengan JSX:
```
Comment (vferdiansyah) [inline content/docs/react-without-jsx.md]:
```suggestion
Misalnya, kode yang ditulis dengan JSX berikut:
```

File: content/docs/react-without-jsx.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ ReactDOM.render(
 );
 ```
 
-can be compiled to this code that does not use JSX:
+Dapat dikompilasi dengan kode dibawah ini untuk yang tidak menggunakan JSX:
```
Comment (vferdiansyah) [inline content/docs/react-without-jsx.md]:
```suggestion
dapat dikompilasi menjadi kode yang tidak menggunakan JSX berikut:
```

File: content/docs/react-without-jsx.md [hunk 5]
```diff
@@ -38,11 +38,11 @@ ReactDOM.render(
 );
 ```
 
-If you're curious to see more examples of how JSX is converted to JavaScript, you can try out [the online Babel compiler](babel://jsx-simple-example).
+Jika Anda ingin tahu lebih lanjut untuk melihat banyak contoh tentang bagaimana JSX dikonversi ke JavaScript, Anda dapat mencobanya di [the online Babel compiler](babel://jsx-simple-example).
```
Comment (vferdiansyah) [inline content/docs/react-without-jsx.md]:
```suggestion
Jika Anda ingin melihat lebih banyak contoh tentang bagaimana JSX dikonversi ke JavaScript, Anda dapat mencobanya di [the online Babel compiler](babel://jsx-simple-example).
```

**Comment by resir014** [comment]
Hi @brilyandk, please address the review items above before I can merge this.

**Pull request merged**